src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +26,21 @@ public void run() {
         }).start();
     }
 
+    private static int sumAllWhenConditionMatched(List<Integer> numbers, Conditional c) {
+        return numbers.stream()
+            .filter(c::isMatched)
```

Comment:
변수 네이밍을 의미있게 하시면 좋을 것 같아요 :)

```suggestion
    private static int sumAllWhenConditionMatched(List<Integer> numbers, Conditional condition) {
        return numbers.stream()
            .filter(condition::isMatched)
```
[clean code - 의미 있는 이름](https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2002%20-%20%EC%9D%98%EB%AF%B8%20%EC%9E%88%EB%8A%94%20%EC%9D%B4%EB%A6%84.md#%EC%9D%98%EB%8F%84%EB%A5%BC-%EB%B6%84%EB%AA%85%ED%9E%88-%EB%B0%9D%ED%98%80%EB%9D%BC)